watson_developer_cloud/text_to_speech_v1.py

```diff
@@ -12,123 +14,1198 @@
+        headers = {'Accept': accept}
         params = {
-            'text': text,
+            'accept': accept2,
             'voice': voice,
-            'format': pronunciation_format
+            'customization_id': customization_id
         }
-        return self.request(method='GET', url='/v1/pronunciation',
-                            params=params, accept_json=True)
+        data = {'text': text}
+        url = '/v1/synthesize'
+        response = self.request(
+            method='POST',
+            url=url,
+            headers=headers,
+            params=params,
+            json=data,
+            accept_json=False)
+        return response
 
-    def customizations(self, language=None):
+    #########################
+    # pronunciation
+    #########################
+
+    def get_pronunciation(self,
+                          text,
+                          voice=None,
+                          format=None,
+                          customization_id=None):
+        """
+        Gets the pronunciation for a word.
+
+        Returns the phonetic pronunciation for the word specified by the `text` parameter.
+        You can request the pronunciation for a specific format. You can also request the
+        pronunciation for a specific voice to see the default translation for the language
+        of that voice or for a specific custom voice model to see the translation for that
+        voice model.   **Note:** This method is currently a beta release.
+
+        :param str text: The word for which the pronunciation is requested.
+        :param str voice: A voice that specifies the language in which the pronunciation is to be returned. All voices for the same language (for example, `en-US`) return the same translation. Retrieve available voices with the `GET /v1/voices` method.
+        :param str format: The phoneme format in which to return the pronunciation. Omit the parameter to obtain the pronunciation in the default format.
+        :param str customization_id: The GUID of a custom voice model for which the pronunciation is to be returned. The language of a specified custom model must match the language of the specified voice. If the word is not defined in the specified custom model, the service returns the default translation for the custom model's language. You must make the request with service credentials created for the instance of the service that owns the custom model. Omit the parameter to see the translation for the specified voice with no customization.
+        :return: A `dict` containing the `Pronunciation` response.
+        :rtype: dict
+        """
+        if text is None:
+            raise ValueError('text must be provided')
         params = {
-            'language': language
-        }
-        return self.request(method='GET', url='/v1/customizations',
-                            params=params, accept_json=True)
-
-    def get_customization(self, customization_id):
```

Comment:
It will break compatibility to remove these methods, and I think that's a problem.  Why not leave them?  I don't think they conflict with any of the generated methods.  Maybe we can mark them as deprecated, and add comments recommending the new method names.  That will set us up to remove them in the next major release.